js/sheetsee.js

```diff
@@ -14,14 +14,17 @@ function initiateTableFilter(data, filterDiv, tableDiv) {
   })
   $(filterDiv).keyup(function(e) {
     var text = $(e.target).val()
-    searchTable(data, text, tableDiv)
+    searchTable(data, text.toLowerCase(), tableDiv)
   })
 }
 
 function searchTable(data, searchTerm, tableDiv) {
   var filteredList = []
   data.forEach(function(object) {
-    var stringObject = JSON.stringify(object).toLowerCase()
+    // Original strategy: Match against the whole object, stringified, for filtering
+    // var stringObject = JSON.stringify(object).toLowerCase()
+    // New strategy: Match against the airline name and sizes
+    var stringObject = object.airline.toLowerCase() + " " + object.sizeinches.toLowerCase() + " " + object.sizecm.toLowerCase()
```

Comment:
I line-broke these all nicely and then forgot to save before committing. Bummer.